<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationWidget</class>
 <widget class="QWidget" name="SegmentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Pixel Classifier</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="FileEditWidget" name="filenameEdit" native="true"/>
      </item>
      <item>
       <widget class="FeatureSelector" name="featureSelector" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="chkRemoveSmallObj">
          <property name="text">
           <string>Remove Small Objects</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="minObjSizeSpinner">
          <property name="specialValueText">
           <string>Min Object Size Px</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Tree depth</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="numMaxDepthSpinner"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnTrain">
            <property name="text">
             <string>Train</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Number of trees</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="numTreesSpinner"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnSegment">
            <property name="text">
             <string>Segment</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>259</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileEditWidget</class>
   <extends>QWidget</extends>
   <header>napari_tracking_analysis.segmentation_widget.segmentation_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FeatureSelector</class>
   <extends>QWidget</extends>
   <header>napari_tracking_analysis.segmentation_widget.segmentation_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
